javascript.md

```diff
@@ -63,13 +63,12 @@ JavaScript CodeStyle
     * only once in the current scope;
     * on a new line;
     * as close as possible to the place where it's first used.
-
-  * Each `var` statement should have only one variable declared in it.
+  * Implied global variables should not be used.
 
 **Good:**
 ```javascript
-var keys = ['foo', 'bar'];
-var values = [23, 42];
+var keys = ['foo', 'bar'],
```

Comment:
Looks like var-holywar is cornerstone and crucial for some other code style rules. 
Is there any chance we all agreed on some explicit var-statement rule? Moreover, it's inconvenient to have whole codestyle without any mention on how to arrange var-block. All other popular codestyles (Crockford's, airbnb, Google's) have that rule.